roles/taca/templates/site_taca_ont_delivery.yml.j2

```diff
@@ -0,0 +1,15 @@
+log:
+    file: "{{ ngi_pipeline_path }}/log/taca.log"
+
+organise:
+  incoming_path: "{{ proj_root }}/{{ uppmax_project }}/incoming"
+  nanopore_path: "{{ proj_root }}/{{ uppmax_project }}/nobackup/NGI/ONT_TAR"
+  illumina_path: "{{ proj_root }}/{{ uppmax_project }}/nobackup/NGI/DATA"
+  element_path: "{{ proj_root }}/{{ uppmax_project }}/nobackup/NGI/DATA"
```

Comment:
Do we need the illumina path and element path in ont_delivery.yml?

Reply:
Ah, yes this will be for Illumina and element later on. I should rename the file I think 😅